txeventq/kafka-connector-example/sql/setupTeq.sql

```diff
@@ -0,0 +1,20 @@
+set serveroutput on
+
+alter pluggable database all close immediate;
+drop pluggable database cdb1_pdb1 including datafiles;
+create pluggable database cdb1_pdb1 admin user &1 identified by &2 file_name_convert=('&3', 'pdb1');
+alter pluggable database all open;
```

Comment:
Same as above, why would we open all PDBs? Let's just open the on that we have created